gen/calc.py

```diff
@@ -587,6 +592,7 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
         'mesos_dns_ip_sources': '["host", "netinfo"]',
         'master_external_loadbalancer': '',
         'mesos_log_retention_mb': '4000',
+        'mesos_container_log_destination': 'both',
```

Comment:
if logs are not found in journald, the CLI will fallback to sandbox, so we should be good.